utilities/convert_frosst/smirffishFrcmod.parm99Frosst.txt

```diff
@@ -361,14 +363,15 @@ DIHE
 #
 # C-O bonds
 # O-Csp3
-[*:1]-[#6X4:2]-[#8X2H1:3]-[#1:4]     1  0.160    0.0  -3.0  parm99 X_-CT-OH-X  JCC,7,(1986),230
-[*:1]-[#6X4:2]-[#8X2H1:3]-[#1:4]     1  0.250    0.0   1.0  parm99 HO-OH-CT-CT  Junmei et al, 1999
-[*:1]-[#6X4:2]-[#8X2H0:3]-[*:4]      1  0.383    0.0  -3.0  parm99 CT-CT-OS-CT 
-[*:1]-[#6X4:2]-[#8X2H0:3]-[*:4]      1  0.100  180.0   2.0  parm99 CT-CT-OS-CT 
```

Comment:
Ah, right, no, this was not a deletion, just a change. For the record, on the lines 365, 366, and 367 from above I changed the wildcard in the first position to #6X4, and the wildcard in the last for 366 and 367 to #6X4.

Anyway, line 365 was definitely a bugfix, @bannanc - if you look at it, it overrides line 364 EXACTLY. Christopher appears to have just duplicated the SMIRKS pattern from the previous line and then never corrected it to match the specified pattern (CT-CT-OH-HO). The other two result in parameter inconsistencies with parm@frosst when applied to AlkEthOH because a generic torsion is otherwise applied in that case. There's nothing in his files which indicates he made this choice deliberately, so I'm interpreting it as an overly broad CT-CT-OS-CT (i.e. I'd want to see at least carbon in the first and last positions; also, normally he placed generics at the top of sections, so it would be out of place to have a generic here when it's preceded by more specialized torsions already).

Another argument in favor of my view is that I think we want this force field to be like parm@frosst except when we're removing unnecessarily specialization. The issues corrected on lines 366 and 367 are in the opposite direction -- Christopher had (accidentally, I believe) added additional specialization not present in parm@Frosst.

I'll try and discuss it with him the next time we talk, but I'm pretty confident in these changes. :)